nvdaHelper/remote/sysListView32.cpp

```diff
@@ -30,7 +30,10 @@ error_status_t nvdaInProcUtils_sysListView32_getGroupInfo(handle_t bindingHandle
 	group.cbSize=sizeof(group);
 	group.mask=LVGF_HEADER|LVGF_FOOTER|LVGF_STATE;
 	group.stateMask=0xffffffff;
-	if(!SendMessage((HWND)UlongToHandle(windowHandle),LVM_GETGROUPINFOBYINDEX,(WPARAM)groupIndex,(LPARAM)&group)) {
+	// ListView_GetGroupInfoByIndex macro has no return value, using SendMessage directly so errors are caught.
+	// The meaning of the return value depends on the message sent, for LVM_GETGROUPINFOBYINDEX,
+	// it returns TRUE if successful, or FALSE otherwise.
+	if(!SendMessage(static_cast<HWND>(UlongToHandle(windowHandle)),LVM_GETGROUPINFOBYINDEX,(WPARAM)groupIndex,(LPARAM)&group)) {

```

Comment:
There are still c-style casts in here. Moving the call out of the `if` condition and putting the arguments onto individual lines makes it a little easier to catch.

```suggestion
	const auto sendMsgRes = SendMessage(
		static_cast<HWND>(UlongToHandle(windowHandle)),
		LVM_GETGROUPINFOBYINDEX,
		static_cast<WPARAM>(groupIndex),
		reinterpret_cast<LPARAM>(&group)
	);
	if(TRUE != sendMsgRes) {

```

Reply:
AH, I'm sorry. I was pretty sure I addressed this, probably never committed.